Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -354,7 +356,7 @@ public IEnumerable<string> ListFiles(string path, SearchOption searchOption, par
 
             using (var repo = new LibGit2Sharp.Repository(RepositoryPath))
             {
-                var files = repo.Diff.Compare<TreeChanges>(null, DiffTargets.Index, lookupList, compareOptions: new CompareOptions() { IncludeUnmodified = true, Similarity = SimilarityOptions.None })
+                var files = repo.Diff.Compare<TreeChanges>(null, DiffTargets.Index, lookupList, new ExplicitPathsOptions { ShouldFailOnUnmatchedPath = false }, compareOptions: new CompareOptions() { IncludeUnmodified = true, Similarity = SimilarityOptions.None })
```

Comment:
not sure what `ExplicitPathsOptions`, can't seem to find any docs for it. most the wikis on their repo are outdated.